src/components/Cities.js

```diff
@@ -0,0 +1,56 @@
+import React, { useEffect, useState } from 'react';
+import {
+  Loader, Text, Image,
+} from '@mantine/core';
+import { useSelector } from 'react-redux';
+import CityItem from './CityItem';
+import './cities.css';
+
+const Cities = () => {
+  const [loading, setLoading] = useState(true);
+  useEffect(() => {
+    setTimeout(() => {
+      setLoading(false);
+    }, 500);
+  }, []);
+
+  const { cities } = useSelector((state) => state.cities);
+
+  return (
+    <div className="cities-container">
+      {/* header status bar */}
+      <div className="cities-status-bar">
+        <Text className="city-header-title">My Weather App</Text>
+      </div>
+
+      {/* sub-header */}
+      <div className="sub-header-city">
+        <Image
+          radius="md"
+          src="https://cdn2.iconfinder.com/data/icons/weather-flat-14/64/weather02-512.png"
+          alt="Weather icon"
+          width={80}
+        />
+        <Text className="weather-desc-title">Weather App</Text>
+      </div>
+
```

Comment:
- [ ] Great job working on some of the changes suggested by the previous reviewer. Your design is now looking like the given [Nelson Sakwa design](https://www.behance.net/gallery/31579789/Ballhead-App-%28Free-PSDs%29). Unforf tunately the previous reviewer spoke of select box with selection not working, but right now there is even no search box for any `filter`. 
![filter](https://user-images.githubusercontent.com/77069011/219906035-2ec5097a-d5e4-4668-a997-59aaa426a28d.png) 
See screenshot below, there is no search bar to filter maybe by city name. Kindly review this. 
![Screen Shot 2023-02-25 at 8 13 12 AM](https://user-images.githubusercontent.com/62849872/221367493-da717be4-592b-45b1-b080-de159f1042fb.png)

- [ ] When I click on a city to get details about the city, your design presents some weird dots and brackets which is not found on the given design. Kindly get this cleaned. See screenshot below;
![Screen Shot 2023-02-25 at 8 16 03 AM](https://user-images.githubusercontent.com/62849872/221367770-f2bc12f4-8629-4e7a-9702-5804f709c430.png)